src/actions/vstudio/_vstudio.lua

```diff
@@ -340,7 +340,13 @@
 		-- Then the system libraries, which come undecorated
 		local system = config.getlinks(cfg, "system", "fullpath")
 		for i = 1, #system do
-			table.insert(links, path.appendextension(system[i], ".lib"))
+			-- Add extension if required
+			local link = system[i]
+			if link:match("[^.]+$") == link then
```

Comment:
I thought I did a quick test showing the opposite. Maybe VS regex behaves different than Lua's... It'd be nice to add the unit test showing it works for multiple dot's.